src/components/panel/demoGroups/index.html

```diff
@@ -0,0 +1,46 @@
+<div ng-controller="PanelGroupsCtrl as ctrl" ng-cloak>
+
+  <md-toolbar class="md-accent">
+    <div class="md-toolbar-tools">
+      <md-button
+        class="md-icon-button"
+        aria-label="Settings"
+        ng-click="ctrl.showMenu($event, ctrl.settings, 'toolbar')">
```

Comment:
'toolbar' is still here. It should be written in the `showMenu` method, not passed in.